lib/config.js

```diff
@@ -48,17 +48,11 @@ config.views.system.config = {
   packageConfigPaths: [
     '/modules/*/package.json'
   ],
-  packages: {
-    angular: {
-      main: './angular.js',
-      format: 'global',
-      defaultExtension: 'js'
-    }
-  },
+  packages: {},
   transpiler: 'plugin-babel'
 };
 // packages to ignore in import all
-config.views.system.importAllIgnore = ['angular'];
+config.views.system.importAllIgnore = ['jquery', 'bootstrap'];
```

Comment:
Should not be hard-coded or even a default ... so need to figure out a way around this.